drivers/vertica/vertica.go

```diff
@@ -30,6 +79,35 @@ func init() {
 			}
 			return ver, nil
 		},
+		Open: func(_ context.Context, u *dburl.URL, stdout, stderr func() io.Writer) (func(string, string) (*sql.DB, error), error) {
+			return func(_ string, params string) (*sql.DB, error) {
```

Comment:
Nit: since there are two values being returned here, extract this function into a named one, to make this more readable.

Reply:
I believe I cannot extract nested function form here because it uses variable **u *dburl.URL** from local scope.